datashader/__init__.py

```diff
@@ -14,3 +14,10 @@
     from .dask import *
 except ImportError:
     pass
+
+try:
+    def test():
+        import os, pytest
+        pytest.main(os.path.dirname(__file__))
+except ImportError:
+    pass
```

Comment:
It won't be a hard requirement, the call to `test` will just fail when called with `ImportError`. The `try` block is only testing if a function can be created, which won't ever fail with `ImportError`.